dataPipelines/gc_thumbnails/cli.py

```diff
@@ -0,0 +1,40 @@
+import click
+
+from .utils import ThumbnailsCreator
+import typing as t
+
+####
+# CLI
+####
+
+@click.group()
+def cli():
+    pass
+
+
+@cli.command(name='process')
+@click.option(
+    '-f',
+    '--file-name',
```

Comment:
Might be better to rename this arg and its' downstream analogues to `--file-path / file_path` or just `--file / file`, since the arg is supposed to be a path to a file rather than a name. Same for the help text - might be better to say it's a path to input file rather than a name.

Reply:
renamed to --output-directory and --input-directory, for now